src/Chapter2.hs

```diff
@@ -136,34 +136,34 @@ functions in GHCi and insert the corresponding resulting output below:
 
 List of booleans:
 >>> :t [True, False]
-
+[True, False] :: [Bool]
```

Comment:
I see that `doctest` tests are failing on CI for this Chapter:
* https://github.com/poormate/learn4haskell/runs/1199106040

Unfortunately, `doctest` is not smart enough to check only the first line, so you need to add empty line after each answer. Sorry for the inconvenience :disappointed: We've already patched this task in `master` to add empty lines after each test in this task.
```suggestion
[True, False] :: [Bool]

```